### PR TITLE
solution to hw21

### PR DESCRIPTION
**Pull request by syevtushok**

### CODE REVIEW

File: src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+    public ItemDaoJdbcImpl(Connection connection) {
+        super(connection);
+    }
+
+    @Override
+    public Item create(Item item) {
+        String name = item.getName();
+        BigDecimal price = item.getPrice();
+        String query = "INSERT INTO " + DB_NAME + ".items (name, price) "
```
Comment (Tarabarnik) [inline src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java]: try to use String.format in all methods

File: src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+                + "VALUES('" + name + "', " + price + ")";
+        try (Statement statement = connection.createStatement()) {
+            statement.execute(query);
+        } catch (SQLException e) {
+            logger.error("Cannot create item\n" + e);
```
Comment (Tarabarnik) [inline src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java]:
```suggestion
            logger.error("Cannot create item", e);
```
optional, but I prefer this :)

File: src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+    }
+
+    @Override
+    public Item create(Item item) {
+        String name = item.getName();
+        BigDecimal price = item.getPrice();
+        String query = "INSERT INTO " + DB_NAME + ".items (name, price) "
+                + "VALUES('" + name + "', " + price + ")";
+        try (Statement statement = connection.createStatement()) {
+            statement.execute(query);
```
Comment (Tarabarnik) [inline src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java]:
```suggestion
            statement.executeUpdate(query);
```

Comment (syevtushok) [reply src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java]: done

File: src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java (hunk 2 above)
Comment (syevtushok) [reply src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java]: done

File: src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java (hunk 1 above)
Comment (syevtushok) [reply src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java]: done

**Review by Tarabarnik — APPROVED** [review]

**Pull request merged**